engine/src/flutter/impeller/renderer/backend/gles/capabilities_gles.cc

```diff
@@ -235,4 +235,8 @@ ISize CapabilitiesGLES::GetMaximumRenderPassAttachmentSize() const {
   return max_texture_size;
 }
 
+size_t CapabilitiesGLES::GetMinimumUniformAlignment() const {
+  return 256;
```

Comment:
You probably meant to do this in a followup patch. But if not, the general idea is an ivar initialized only on ES 3.0 (where UBO use is possible) with the value of [`glGet*(GL_UNIFORM_BUFFER_OFFSET_ALIGNMENT)`](https://registry.khronos.org/OpenGL-Refpages/es3.0/html/glGet.xhtml) and `1u` elsewhere since the uniform uploads are by copy with `glUniform*`.